Visual Studio extension doesn't build anymore

It seems that building the Visual Studio Extension for VS2022 fails, likely due to recent changes within the ```Microsoft.VisualStudio.SDK``` package which tries to resolve a sub package of the same version that doesn't exist ```Microsoft.VisualStudio.SDK.Analyzers```.

> D:\a\MonoGame\MonoGame\Templates\MonoGame.Templates.VSExtension\MonoGame.Templates.VSExtension.csproj : error NU1102: Unable to find package Microsoft.VisualStudio.SDK.Analyzers with version (>= 17.7.20)
D:\a\MonoGame\MonoGame\Templates\MonoGame.Templates.VSExtension\MonoGame.Templates.VSExtension.csproj : error NU1102:   - Found 26 version(s) in nuget.org [ Nearest version: 16.10.10 ]
D:\a\MonoGame\MonoGame\Templates\MonoGame.Templates.VSExtension\MonoGame.Templates.VSExtension.csproj : error NU1102:   - Found 0 version(s) in Microsoft Visual Studio Offline Packages

Investigations would be welcome to fix the build script.

--- a/Templates/MonoGame.Templates.VSExtension/MonoGame.Templates.VSExtension.csproj
+++ b/Templates/MonoGame.Templates.VSExtension/MonoGame.Templates.VSExtension.csproj
@@ -59,7 +59,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.*" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.*" />
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.9.*" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="Templates.pkgdef">

